kolibri/plugins/coach/assets/src/views/lessons/LessonSummaryPage/sidePanels/LessonResourceSelection.vue

```diff
@@ -160,6 +192,10 @@
         context:
           "In the 'Manage lesson resources' coaches can add new/remove resource material to a lesson.",
       },
+      searchFiltersTitle: {
+        message: 'Search',
+        context: 'TO DO',
```

Comment:
we can use the `searchLabel` in `commonCoreStrings` for this, rather than add a new string here. 

the **context** is a message that appears in the GUI of the app we use for translators who are translating Kolibri, and it helps them understand the, well, context, of the string! Sometimes Radina adds these in, but you can also take a "first attempt" for any strings that you do add (and she can edit them later). Details about where the user might encounter the text, or how they're interacting with it, can be a good place to start. 

here's an example: 

```
searchForUser: {
    message: 'Search for a user...',
    context:
      'Text which appears in the search field above the table with users from whom to choose from (e.g. when enrolling learners to a class, selecting users to sync, etc.)',
  },
  ```